docs/requirements/requirements.md

```diff
@@ -205,22 +232,83 @@ This document is in the making since we are still in the requirements engineerin
 - There should be a kind of demo mode so that a new user can try out the system quickly and easily: We provide test data with which the system can be directly *loaded*.
 - Origin: Developers
 
+#### R-U-015: Repeatability of experiments
+
+- The system must ensure that researchers can repeat experiments in a controlled environment as often as they wish.
+- As a researcher, I want to repeat an experminent multiple times and I need the environment to be the exact same for each repetition of the experiment.
+- A stakeholder said: "If I have scaling in place, when the experiment is finished, you want the state to be just like before the experiment. This allows repeatability."
+- This requirement could also be moved into the section Configurations.
+- Origin: Researcher *Toni*
+
+#### R-U-016: Scheduling of experiments
+
+- The system must offer the possibility to schedule experiments.
+- As a researcher, I want to be able to schedule experiments.
+- This requirement could also be moved into the section Configurations.
+- Origin: Researcher *Toni*
+
 ### Non-functional Requirements
 
 #### R-NFR-001: Performance
 
 - The microservices must be performant.
 - Origin: Examiner
 
-#### R-NFR-002: Elasticity
+#### R-NFR-002: Elasticity / Auto scaling
 
 - The system must be *elastic*.
-- Origin: Examiner
+- The system must scale automatically.
+- Origin: Examiner, Researcher *Toni*
 
 ##### Open Question
 
 - What exactly does *elastic* mean? What is the examiner's understanding of the term?
 
+#### R-NFR-003: Resilience
+
+- The system should be resilient.
+- Origin: Researcher *Toni*
+
+### Configurations
+
+See also R-U-003.
+
+#### R-Conf-001: Variable in size
+
+- The system's size should be configurable.
+- As a researcher, I want to be able to decide how big or small the system actually is. I want to set the system's size in order to have it fit my needs.
+- Origin: Researcher *Toni*
+
+#### R-Conf-002: Load profiles
+
+- The system must offer the possibility to use load profiles.
+- As a researcher, I want to specify load profiles to control the system when experimenting with it.
+- Advice from stakeholder: Derive the load profiles from the domain.
+- As a researcher, I want diversity in the system's load patterns because I want to know, if I deploy an elasticity policy, how will that perform?
+- Origin: Researcher *Toni*
+
+#### R-Conf-003: Elasticity policies
+
+- The system must offer the possibility to specify elasticity policies.
+- See R-Conf-002
+- Origin: Researcher *Toni*
+
+#### R-Conf-004: Scaling policies
+
+- The system should offer different alternatives for scaling.
+- The system should allow the user to configure the scaling techniques in place.
+- Origin: Researcher *Toni*
+
+##### Open Question
+
+- Is this basically requirement R-Conf-003?
+
+#### R-Conf-005: System variants
+
+- The system should have different variants.
+- e.g., one variant with database as a service, another variant with a shared database
```

Comment:
Which variants could be options except the database example?

Reply:
Good question, Idk right now. Of course, we could try to come up with something. The stakeholder who mentioned this only had this one example but he was insisting that variants would be great. I will add this as an open question to the requirement.